cli/azd/pkg/convert/util.go

```diff
@@ -14,3 +14,12 @@ func ToValueWithDefault[T any](ptr *T, defaultValue T) T {
 func RefOf[T any](value T) *T {
 	return &value
 }
+
+// Attempts to convert the specified value to a string, otherwise returns the default value
+func ToStringWithDefault(value any, defaultValue string) string {
```

Comment:
I think this will panic if value isn't a string...

Should be this?

```
func ToStringWithDefault(value interface{}, defaultValue string) string {
	if value == nil {
		return defaultValue
	}

	str, ok := value.(string)
	if !ok {
		return defaultValue
	}

	return str
}

```

Reply:
Or

```
func ToStringWithDefault(value interface{}, defaultValue string) string {
	if str, ok := value.(string); ok {
		return str
	}
	return defaultValue
}

```